OWC/OrificeModel/wecSimInputFile.m

```diff
@@ -1,35 +1,33 @@
 %% Simulation Data
 simu = simulationClass();               % Initialize Simulation Class
-simu.simMechanicsFile = 'GBM1.slx';     % Specify Simulink Model File
+simu.simMechanicsFile = 'OWC_GBM.slx';  % Specify Simulink Model File
 simu.mode = 'normal';                   % Specify Simulation Mode ('normal','accelerator','rapid-accelerator')
 simu.explorer = 'on';                   % Turn SimMechanics Explorer (on/off)
 simu.startTime = 0;                     % Simulation Start Time [s]
 simu.rampTime = 10;                   	% Wave Ramp Time [s]
 simu.endTime = 130;                     % Simulation End Time [s]
+% simu.solver = 'ode45';                  % simu.solver = 'ode4' for fixed step & simu.solver = 'ode45' for variable step 
 simu.solver = 'ode23t';                 % simu.solver = 'ode4' for fixed step & simu.solver = 'ode45' for variable step 
 simu.dt = 0.005; 						% Simulation time-step [s]
 simu.cicEndTime = 15;
```

Comment:
I was unable to run this simulation. I attempted to run different solvers, different time-steps, but nothing worked. I got a convergence error each time.